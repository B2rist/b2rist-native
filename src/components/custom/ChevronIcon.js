import React from 'react';
import {IconButton} from 'react-native-paper';

const ChevronIcon = props => <IconButton {...props} icon="chevron-right" />;

export default ChevronIcon;
