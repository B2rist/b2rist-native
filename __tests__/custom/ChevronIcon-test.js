import 'react-native';
import React from 'react';
import ChevronIcon from 'components/custom/ChevronIcon';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('chevron renders correctly', () => {
  const chevron = renderer.create(<ChevronIcon />).toJSON();
  expect(chevron).toMatchSnapshot();
});
