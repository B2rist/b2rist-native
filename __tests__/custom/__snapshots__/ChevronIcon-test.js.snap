// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`chevron renders correctly 1`] = `
<View
  collapsable={false}
  style={
    {
      "backgroundColor": undefined,
      "borderColor": undefined,
      "borderRadius": 18,
      "borderWidth": 0,
      "elevation": 0,
      "height": 36,
      "margin": 6,
      "overflow": "hidden",
      "width": 36,
    }
  }
>
  <View
    accessibilityComponentType="button"
    accessibilityRole="button"
    accessibilityState={
      {
        "disabled": true,
      }
    }
    accessibilityTraits="button"
    accessible={true}
    centered={true}
    collapsable={false}
    focusable={true}
    hitSlop={
      {
        "bottom": 6,
        "left": 6,
        "right": 6,
        "top": 6,
      }
    }
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      [
        {
          "overflow": "hidden",
        },
        false,
        [
          {
            "alignItems": "center",
            "flexGrow": 1,
            "justifyContent": "center",
          },
          {
            "borderRadius": 18,
          },
        ],
      ]
    }
  >
    <Text
      accessibilityElementsHidden={true}
      allowFontScaling={false}
      importantForAccessibility="no-hide-descendants"
      pointerEvents="none"
      selectable={false}
      style={
        [
          {
            "color": "rgba(73, 69, 79, 1)",
            "fontSize": 24,
          },
          [
            {
              "lineHeight": 24,
              "transform": [
                {
                  "scaleX": 1,
                },
              ],
            },
            {
              "backgroundColor": "transparent",
            },
          ],
          {
            "fontFamily": "Material Design Icons",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          {},
        ]
      }
    >
      ó°…‚
    </Text>
  </View>
</View>
`;
